src/lib.rs

```diff
@@ -480,35 +479,16 @@ impl_as_byte_slice_arrays!(!div 4096, N,N,N,N,N,N,N,);
 /// [`OsRng`]: rngs::OsRng
 #[cfg(feature="std")]
 pub trait FromEntropy: SeedableRng {
```

Comment:
Do we want to deprecate this trait as well?

Reply:
That is the question from the top comment (whoops, ~`EntropyRng`~ → `FromEntropy`).